ros/src/tl_detector/sim_traffic_light_config.yaml

```diff
@@ -3,4 +3,13 @@ detection_threshold: 0.3
 detection_iou: 0.45
 min_red_light_size: 4
 light_change_history_length: 2
-red_light_threshold: 0.5
\ No newline at end of file
+red_light_threshold: 0.5
+stop_line_positions:
```

Comment:
what about site_ file ? how it's gonna work there?

Reply:
fixed!